README.md

```diff
@@ -103,7 +103,7 @@ Config Name | Description
 --- | ---
 client_id | Default clientId. MUST be a string
 client_secret | Default clientSecret. MUST be a string
-realm | realm query parameter (for oauth1) added to `authorizationUrl` and `tokenUrl` . MUST be a string
+realm | The OAuth realm parameter for `authorizationUrl` and `tokenUrl`. Optional. If specified it MUST be a string. If a `{realm}` placeholder is present in the URL path, it will be replaced with this value. If no such placeholder is present, the value will be added as the value for querystring parameter 'realm'.
```

Comment:
The old docs were very concise. I added some more detail and mentioned the `{realm}` placeholder and `realm` query parameter explicitly so people (hopefully) understand what it does.